extensions/src/simple_example/index.ts

```diff
@@ -55,30 +55,30 @@ export default class SimpleTypescript extends extension(details, "ui", "customSa
     console.log(value);
   }
 
-  @reporter`Add ${{type: "string", defaultValue: "yee"}} to ${{type: "string", defaultValue: "haw"}}: strings simple`
-    simpleReporterString(x: string, y: string) {
-        return x + y;
-    }
+  @(scratch.reporter`Add ${{ type: "string", defaultValue: "yee" }} to ${{ type: "string", defaultValue: "haw" }}: strings simple`)
```

Comment:
For some reason, typescript is requiring us to wrap the decorator invocation in `(` `)`s, not sure why, and I dont like it!